torch/csrc/jit/runtime/register_cuda_ops.cpp

```diff
@@ -18,19 +18,47 @@ c10::AliasAnalysisKind aliasAnalysisFromSchema() {
 
 RegisterOperators const reg({
     Operator(
-        "cuda::current_stream(int64_t val) -> __torch__.torch.classes.cuda.Stream",
+        "cuda::current_stream.device(Device? device) -> __torch__.torch.classes.cuda.Stream",
         [](Stack* stack) {
-          auto idx = uint16_t(pop(stack).toInt());
-          auto s = c10::cuda::getCurrentCUDAStream(idx);
+          auto device = pop(stack).toOptional<c10::Device>();
+          c10::DeviceIndex device_index = device.has_value()
+              ? device->index()
+              : c10::cuda::current_device();
+          auto s = c10::cuda::getCurrentCUDAStream(device_index);
           auto st = make_custom_class<torch::jit::CUDAStream>(s);
           push(stack, IValue(st));
         },
         aliasAnalysisFromSchema()),
     Operator(
-        "cuda::default_stream(int64_t val) -> __torch__.torch.classes.cuda.Stream",
+        "cuda::current_stream.int(int? val) -> __torch__.torch.classes.cuda.Stream",
         [](Stack* stack) {
-          auto idx = uint16_t(pop(stack).toInt());
-          auto s = c10::cuda::getDefaultCUDAStream(idx);
+          auto idx = pop(stack).toOptional<int64_t>();
+          c10::DeviceIndex device_index =
+              idx.has_value() ? static_cast<c10::DeviceIndex>(idx.value()) : -1;
+          auto s = c10::cuda::getCurrentCUDAStream(device_index);
+          auto st = make_custom_class<torch::jit::CUDAStream>(s);
+          push(stack, IValue(st));
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "cuda::default_stream.device(Device? device) -> __torch__.torch.classes.cuda.Stream",
+        [](Stack* stack) {
+          auto device = pop(stack).toOptional<c10::Device>();
+          c10::DeviceIndex device_index = device.has_value()
+              ? device->index()
+              : c10::cuda::current_device();
```

Comment:
Same here.

Reply:
This is due to clang-formatting.

Reply:
`clang-format` changed the content of the code?

Reply:
Fixed now.